.github/workflows/go.yml

```diff
@@ -44,13 +43,73 @@ jobs:
 
   test:
     runs-on: ubuntu-latest
+    name: Update coverage badge
     steps:
     - uses: actions/checkout@v3
+      with:
+        persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal access token.
+        fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
 
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
         go-version: 1.21
 
-    - name: Test
-      run: go test -v ./...
+    - name: Unit test
+      run: go test -cover -coverprofile coverage.out ./...
```

Comment:
I see the verbose flag was removed.  I kind of liked being able to see the test results in the pipeline output. Was that intentionally removed?

Reply:
> I see the verbose flag was removed. I kind of liked being able to see the individual test output in the pipeline output. Was that intentionally removed?

Unintentional - that accidentally got dropped when playing around with different coverage options and trying to visually see the coverage for each separate module. I'll add it back in.